pkg/notes/themes/testdata/sig-api-machinery-deprecation-warnings.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Deprecation warnings
+sig: api-machinery
+---
+SIG API Machinery implemented warning mechanisms when using deprecated APIs that are visible to API consumers and metrics visible to cluster administrators. Requests to a deprecated API are returned with a warning containing a target removal release and any replacement API.
```

Comment:
Do you think we should use YAML here? We could still write it in plain markdown, but would have to keep the indendation:

```yaml
title: Title
sig: release
content: |
  This is the `markdown` formatted content.
  Which should also work with multiple lines.

Reply:
Yes of course we can do all yaml :)  

My thought behind composing the themes like this was that themes defined here are more document fragments than structured data. So it made sense to give the writer more flexibility when writing more complex entries for the themes (for example with embedded code blocks). The little metadata we have for each could then be stored in the markdown front matter.

Reply:
Ah okay I see. On the other hand with YAML we're able to put multiple themes into one file. Not sure if that's a use case. It seems to already read the files alphabetically which implicitly sets the ordering.